.gitignore

```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /.pnp
+/package-lock.json
```

Comment:
Why ignore `package-lock.json`? :thinking:

Also remove this file since it's unrelated to this PR.